core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosSchedulerUtils.scala

```diff
@@ -17,15 +17,18 @@
 
 package org.apache.spark.scheduler.cluster.mesos
 
-import java.util.List
 import java.util.concurrent.CountDownLatch
+import java.util.{List => JList}
 
 import scala.collection.JavaConversions._
 
-import org.apache.mesos.Protos.{FrameworkInfo, Resource, Status}
-import org.apache.mesos.{MesosSchedulerDriver, Scheduler}
-import org.apache.spark.Logging
+import com.google.common.base.Splitter
+
+import org.apache.mesos.Protos._
+import org.apache.mesos.{MesosSchedulerDriver, Protos, Scheduler}
 import org.apache.spark.util.Utils
+import org.apache.spark.{Logging, SparkContext}
```

Comment:
dont' use the built-in organizer of IntelliJ -- use the plugin that Aaron Davidson wrote:
https://plugins.jetbrains.com/plugin/7350

I know this is confusing and a pain to get right, its in the docs here, but very easy to overlook:
https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports

so far that has always worked for me (but maybe there are cases it gets wrong?).  For now the reviewers just do a lot of nit-picking :/.  There is also an open PR to get this included in the scalastyle checker, so at least you'd get automatic warnings.